install-and-configure/install/getting-started.md

```diff
@@ -39,7 +39,7 @@ Establishing an ingress controller will allow for control of access for your wor
 ### SSO/SAML/RBAC/OIDC
 
 {% hint style="info" %}
-SSO/SAML/RBAC/OIDC are only officially supported on Kubecost Enterprise plans.
```

Comment:
This change is an improvement.

nit: Maybe just say SAML/OIDC? RBAC is really a feature layered on top of a SAML or OIDC implementation.